backend/apps/owasp/management/commands/generate_project_metadata.py

```diff
@@ -0,0 +1,116 @@
+"""A command to generate project metadata from project_key."""
+
+from pathlib import Path
+
+import yaml
+from django.core.management.base import BaseCommand
+from owasp_schema import get_schema
+from owasp_schema.utils.schema_validators import validate_data
+
+from apps.owasp.models.project import Project, ProjectLevel, ProjectType
+
+MIN_PROJECT_TAGS = 3
+
+
+class Command(BaseCommand):
+    help = "Generates and validates project metadata for a given project key."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "project_key",
+            type=str,
+            help="The key of the project to generate the metadata file.",
+        )
+
+    def handle(self, *args, **options):
+        project_key = options["project_key"]
+        self.stdout.write(f"Attempting to process project: {project_key}")
+
+        try:
+            project = Project.objects.get(key=project_key)
+        except Project.DoesNotExist:
+            self.stderr.write(self.style.ERROR(f"Project with key '{project_key}' not found."))
+            return
+
+        metadata_dict = self.map_data_to_schema(project)
+
+        project_schema = get_schema("project")
+        error_message = validate_data(schema=project_schema, data=metadata_dict)
+
+        if error_message:
+            self.stderr.write(self.style.ERROR("Validation FAILED!"))
+            self.stderr.write(f"Reason: {error_message}")
+            return
+        self.stdout.write(self.style.SUCCESS("Validation successful!"))
+
+        self.stdout.write("Writing validated data to file...")
+        output_dir = Path("schema/data")
```

Comment:
`schema` is no longer a valid location. Use `backend/data/schema/<entity_type>/<entity>.owasp.yaml`